src/pages/instances/InstanceOverviewDeviceList.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { isRootDisk } from "util/instanceValidation";
+import { FormDevice } from "util/formDevices";
+import { LxdDeviceValue, LxdGPUDevice } from "types/device";
+import ResourceLink from "components/ResourceLink";
+
+interface Props {
+  device: LxdDeviceValue;
+  instance: LxdInstance;
+}
+
+const InstanceOverviewDeviceList: FC<Props> = ({ device, instance }) => {
+  return (
+    <div className="u-truncate">
+      {device.type == "disk" ? (
+        isRootDisk(device as FormDevice) ? (
+          <div title={`Pool: ${device.pool}`}>{`pool: ${device.pool}`}</div>
+        ) : (
+          <>
+            <div title={`Storage Volume: ${device.source}`}>
```

Comment:
I think displaying the chips inline next to each other without prefixes is fine? Please can you check with @edlerd on this.

Reply:
I think we can use the structure we use in the used by list i.e. `<PoolChip> / <VolumeChip>`

Reply:
see @edlerd comment below about this.